src/test/unit/project-converter_test.ts

```diff
@@ -452,6 +452,80 @@ export const Polymer = function(info) {
       });
     });
 
+    test(
+        'in a package "polymer", renames a root file "/polymer.html" to "polymer-legacy.js"',
+        async () => {
+          setSources({
+            'test.html': `
+          <link rel="import" href="./polymer.html">
```

Comment:
does this also work for `<link rel="import" href="polymer.html">`, or non-sibling files?